tests/testthat/test-shinytest2-tm_front_page.R

```diff
@@ -0,0 +1,48 @@
+test_that("e2e: tm_front_page initializes without errors", {
+  skip_if_too_deep(5)
+  require(shinytest2)
+
+  data <- simple_cdisc_data()
+  data <- within(data, {
+    attr(ADSL, "metadata") <- list("Author" = "NEST team", "data_source" = "synthetic data")
+  })
+
+  app <- TealAppDriver$new(
+    data = data,
+    modules = teal::modules(
```

Comment:
does it need to be covered in `modules`? 
```suggestion
    modules = 
```

Reply:
Not for a single module. I will remove it .

Reply:
also remove the closing bracket

Reply:
done https://github.com/insightsengineering/teal.modules.general/pull/717/commits/8c6be6159dd2a64eb3d2dfbfb90fa4d5078ce6c0